CMakeLists.txt

```diff
@@ -748,15 +745,13 @@ if(BUSTED_LUA_PRG)
     set_target_properties(functionaltest-lua PROPERTIES FOLDER test)
 endif()
 
-if(LUACHECK_PRG)
-  add_custom_target(lualint
-    COMMAND ${LUACHECK_PRG} -q runtime/ scripts/ src/ test/
```

Comment:
The old target used `COMMAND ${LUACHECK_PRG}` and the new target uses `execute_process()`. I wonder if we still get the benefits of cmake's "dirty workspace detection".